### PR TITLE
NIOS 8.6.3 template

### PR DESCRIPTION
**Pull request by nsharda-infoblox**
Changes: 
1. Add zone support for VM 
2. Changes API version
3. Changes image version of 8.6.3

**DO NOT REVIEW THIS PR**

### CODE REVIEW

File: main/vm-availabilityset-existing.json [hunk 1]
```diff
@@ -5,6 +5,9 @@
     "location": {
       "type": "string"
     },
+    "zone": {
+      "type": "string"
```
Comment (kkhanna-infoblox) [inline main/vm-availabilityset-existing.json:9]: did you run AZ templates best practices test suite after these changes?. Hope that there is no warning that zone is not getting used.

File: main/mainTemplate.json [hunk 2]
```diff
@@ -8,7 +8,7 @@
         "artifactsBaseUrl": "/",
         "description": "The base URL for dependent assets"
       },
-      "defaultValue": "https://raw.githubusercontent.com/infobloxopen/infoblox-azure-templates/master/main/"
+      "defaultValue": "https://raw.githubusercontent.com/nsharda-infoblox/infoblox-azure-templates/863-template/main/"
```
Comment (kkhanna-infoblox) [inline main/mainTemplate.json]: modify this to master

**Review by kkhanna-infoblox — CHANGES_REQUESTED** [review]
i hope you have tested both direct templates deployment and marketplace offer

File: main/mainTemplate.json (hunk 2 above)
Comment (nsharda-infoblox) [reply main/mainTemplate.json]: Hi Kapil, yes, template testing is still in progress. Needed this to check the template. Will change back to master.

Will add label of Do No Review for now. I'll update PR when I am finished with testing.

**Review by kkhanna-infoblox — APPROVED** [review]

**Pull request merged**